docs/contributing/releases.md

```diff
@@ -18,6 +18,7 @@ and it is being used in nix environments.
 
 ### prerelease sanity checks
 
+- [ ] set the supported GHC versions and their corresponding cabal project-files in `bindist/ghcs` according to the [GHC version deprecation policy](../supported-versions.md#ghc-version-deprecation-policy)
```

Comment:
I think this is in the wrong section.

Is this really the only new step? Isn't there something that needs to be done with all these new mysterious makefiles?

Reply:
These makefiles are used by gitlab CI to build the new bindists. They will replace the static bindists. I don't really understand the release documentation, so I'd appreciate someone with more knowledge can adjust it.

Reply:
On the other hand, you understand the new stuff better than anyone else :) So if you don't want to integrate it into the existing docs, perhaps you could write a standalone explanation and then we can process that. But I think you've got to write us _something_ :)

Reply:
This will require follow-up PRs anyway to integrate better with the github actions workflows. I've never touched them so far, only the gitlab ones.

The static release builds via github actions will be obsolete and instead release artifacts will only be pulled from gitlab pipelines. That's not automated yet and might be done manually for now.